CONP_FAQ.md

```diff
@@ -1,47 +1,137 @@
 
-5. **I clicked on the download button on the file list, where is the data?**
+#### **I clicked on the download button on the file list, where is the data?**
 
-The download buttons on the list of files downloads metadata for each dataset, in a single `.json` file which contains a detailed description of the data in DATS format.  The data files themselves can be downloaded using DataLad as described on the results page for each individual dataset.
+The download buttons on the list of files downloads metadata for each dataset, in a single `.json` file which contains a detailed description of the data in DATS format.  The data files themselves can be downloaded using DataLad as described on the results page for each individual dataset.
+
+## <a name="data-upload"></a> Data Sharing/Upload
 
-6. **Why should I upload data to CONP?**
+#### **Why should I upload data to CONP?**
 
 Uploading to CONP will increase its visibility and accessibility to other neuroscience researchers in a collaboration-focused, open science environment. 
 
-7. **How do I upload data to CONP?**
+#### **How do I upload data to CONP?**
 
 Data can be uploaded to CONP either through [Zenodo](https://zenodo.org) by tagging your dataset with the keyword `canadian-open-neuroscience-platform`, through [OSF](https://osf.io) by tagging your dataset with `canadian-open-neuroscience-platform` and setting the dataset to `Public`, or manually through DataLad.  [Detailed technical instructions](https://github.com/CONP-PCNO/conp-documentation/blob/master/datalad_dataset_addition_procedure.md) are available for all these options.
 
-8. **To which username/password is DataLad referring to when trying to download via `datalad get` preventad-open or multicenter-phantom datasets?**
+## <a name="tools"></a> Tools & Pipelines
+
+#### **What tools/pipelines are contained in CONP?**
+
+CONP contains a diverse range of XXXXXX tools and pipelines (March 2020).  Many of these tools/pipelines come from neuroscience research institutes, while others link to public resources that may be of interest to neuroscientists. A full list can be found [here](https://portal.conp.ca/pipelines).
+
+#### **What are the requirements to install and run a tool?**
+
+CONP pipelines can be easily installed and run on any computer with a container engine. You will need to install:
+
+- [Docker](https://www.docker.com/) or [Singularity](https://singularity.lbl.gov/) 
+- [Boutiques](https://pypi.org/project/boutiques/) python package (`pip install boutiques`)
+
+#### **How can I run a CONP tool/pipeline?**
+
+First, ensure your have [Boutiques](https://pypi.org/project/boutiques/) installed with [Docker](https://www.docker.com/) or [Singularity](https://singularity.lbl.gov/).
+
+If you have never used Boutiques in the past, ensure to go through the following [tutorial](https://nbviewer.jupyter.org/github/boutiques/tutorial/blob/master/notebooks/boutiques-tutorial.ipynb#reusing_tools).
```

Comment:
"ensure to go through" is awkward, I would suggest "we recommend the following tutorial".